src/class-wpml-cornerstone-update-translation.php

```diff
@@ -1,12 +1,14 @@
 <?php
 
+use WPML\PB\Cornerstone\Utils;
+
 class WPML_Cornerstone_Update_Translation extends WPML_Page_Builders_Update_Translation {
 
 	/** @param array $data_array */
 	public function update_strings_in_modules( array &$data_array ) {
 		foreach ( $data_array as $key => &$data ) {
-			if ( isset( $data['_type'] ) && ! in_array( $data['_type'], array( 'section', 'column', 'row' ) ) ) {
-				$data = $this->update_strings_in_node( $this->get_node_id( $data ), $data );
+			if ( isset( $data['_type'] ) && ! Utils::type_is_layout( $data['_type'] ) ) {
```

Comment:
Can you create or adjust a test for this change @dgwatkins?

Reply:
Done